### ISSUE TITLE
Make builtins more discoverable

### ISSUE THREAD

**Issue by calebzulawski** [CACI-International/ergo#4]
I think an easy way to do this would be to make a default module that contains all of the builtins.  We could name it something like `builtin`, and have it as a builtin itself.  Alternatively, it doesn't need to be named at all, and can just be used for building documentation.

**Comment by afranchuk** [comment]
Yes! I had considered doing this in the past. Another possible name is `prelude`, as that is used in some other languages (but `builtin` is more obvious). As part of the GC additions I may be reworking how the builtins interact with module loading (as part of adding the GC tracing), so I'll keep this in mind.
